### ISSUE TITLE
Document existing models on ivadomed organization

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#657]
We currently have a couple of models under the [ivadomed organization](https://github.com/ivadomed): these are usually hosted in repository with name starting with "model_", with some exceptions (#548).

Currently, there is no "human readable" information in these repositories. It would be nice to at least add a README and an image that illustrates what this model does (task) and for what type of images it is. 

I've started a prototype for [the mice_uqueensland_gm model](https://github.com/ivadomed/mice_uqueensland_gm), feedback welcome 🙂.

Related to https://github.com/ivadomed/ivadomed/issues/344, https://github.com/ivadomed/ivadomed/issues/182

**Comment by andreanne-lemay** [comment]
The prototype is great! We could actually just write the command: `sct_deepseg -i NIFTI_IMAGE -task seg_mice_gm`. The installation will automatically be done if the model is not already downloaded.

**Comment by jcohenadad** [comment]
> The prototype is great! We could actually just write the command: `sct_deepseg -i NIFTI_IMAGE -task seg_mice_gm`. The installation will automatically be done if the model is not already downloaded.

Good point! The README is now updated (https://github.com/ivadomed/mice_uqueensland_gm/blob/master/README.md)
